### ISSUE TITLE
MriModule Validation Loss Logged Incorrectly

### ISSUE THREAD

**Issue by zmhh** [facebookresearch/fastMRI#108]
Steps to reproduce:

- Run "train_unet_demo.py" in fastmri/fastmri_examples/unet
  - Preferably with a specified data folder that only has a single data point for train and val, because it is going to error after a full training and validation epoch
- Observe the following error:

`pytorch_lightning.utilities.exceptions.MisconfigurationException: ModelCheckpoint(monitor='val_loss') not found in the returned metrics: ['loss', 'validation_loss', 'val_metrics/nmse', 'val_metrics/ssim', 'val_metrics/psnr']. HINT: Did you call self.log('val_loss', tensor) in the LightningModule?`

The issue appears to be because MriModule was changed to log `'validation_loss'` instead of `'val_loss'` in the method `validation_epoch_end`. Renaming it back to `'val_loss'` fixes the issue. The change happened in [this commit](https://github.com/facebookresearch/fastMRI/commit/cc42c7a5910bf0e8ce23e1d29d429d17d1e70c78).

**Comment by mmuckley** [comment]
Hello, thanks for finding this. I've opened a PR that fixes the demo scripts. Let me know if it fixes your issue.

[PR 109](https://github.com/facebookresearch/fastMRI/pull/109)

This change made due to a [Pytorch Lightning issue](https://github.com/PyTorchLightning/pytorch-lightning/issues/4553) where using `val_loss` as a metric would overwrite `ModelCheckpoint` behavior with `monitor=None`.  Basically, if you want to log just the most recent epoch (which you need for a leaderboard submission training with validation and training splits together), you can't have `val_loss` as a metric.

**Comment by zmhh** [comment]
Hey Matthew, thanks for the quick response. I have not tested it yet but that makes sense and it looks like it will work. I can probably test changing the monitor param later tonight.

**Comment by mmuckley** [comment]
Okay, I will wait until tomorrow for news of your test and merge once you can confirm.

**Comment by zmhh** [comment]
Hello, yes that fixed the issue. Thanks!

**Comment by mmuckley** [comment]
Okay it's merged.
